packages/eui-theme-borealis/src/eui_theme_borealis_dark.json

```
@@ -36,78 +36,78 @@
   "euiButtonColorDisabledText": "#6A7FA0",
   "euiButtonColorGhostDisabled": "#1D2A3E",
   "euiButtonTypes": {
-    "primary": "#599DFF",
-    "accent": "#ED6BA2",
-    "accentSecondary": "#00BEB8",
-    "success": "#23BE8F",
-    "warning": "#FCD279",
-    "danger": "#F66D64",
+    "primary": "#61A2FF",
+    "accent": "#EE72A6",
+    "accentSecondary": "#16C5C0",
+    "success": "#24C292",
+    "warning": "#FCD883",
```

On the account of "[Remap yellow fill - Looks too desaturated](https://www.figma.com/design/FHys7gLzyvD1gc9DrJM6D8/%5BVR%5D-Elastic-UI-New-Library-%5BTEST%5D?node-id=7656-3427&node-type=frame&t=kgmPMaYNVwRNb97d-0)", shouldn't this be `#EAAE01`?

```suggestion
    "warning": "#EAAE01",
```